<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>324</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_username">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>40</y>
     <width>171</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_password">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>90</y>
     <width>171</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_username">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>101</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>用户名(&amp;U)：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="buddy">
    <cstring>lineEdit_username</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="label_password">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>101</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>密码(&amp;P)：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="buddy">
    <cstring>lineEdit_password</cstring>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_clear">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>250</y>
     <width>89</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: red
</string>
   </property>
   <property name="text">
    <string>清 空</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_confirm">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>250</y>
     <width>89</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: green</string>
   </property>
   <property name="text">
    <string>确 认</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_MAC">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>190</y>
     <width>91</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&amp;MAC：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="buddy">
    <cstring>lineEdit_MAC</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_MAC">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>190</y>
     <width>171</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_NickName">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>101</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>昵称(&amp;N)：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="buddy">
    <cstring>lineEdit_NickName</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_NickName">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>140</y>
     <width>171</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineEdit_username</tabstop>
  <tabstop>lineEdit_password</tabstop>
  <tabstop>lineEdit_NickName</tabstop>
  <tabstop>lineEdit_MAC</tabstop>
  <tabstop>pushButton_confirm</tabstop>
  <tabstop>pushButton_clear</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
